src/basics/Prism.js

```diff
@@ -1,40 +1,91 @@
 import { createGlobalStyle } from "styled-components";
+import Prism from "prismjs";
 
+import "prismjs/plugins/line-numbers/prism-line-numbers";
+import "prismjs/components/prism-bash";
+import "prismjs/components/prism-c";
+import "prismjs/components/prism-cpp";
+import "prismjs/components/prism-css";
+import "prismjs/components/prism-go";
+import "prismjs/components/prism-java";
+import "prismjs/components/prism-json";
+import "prismjs/components/prism-jsx";
+import "prismjs/components/prism-markup";
+import "prismjs/components/prism-python";
+import "prismjs/components/prism-scss";
+import "prismjs/components/prism-toml";
+import "prismjs/components/prism-tsx";
+import "prismjs/components/prism-typescript";
+import "prismjs/components/prism-yaml";
 import "prismjs/plugins/line-numbers/prism-line-numbers.css";
 
 import { PALETTE, FONT_WEIGHT, FONT_FAMILY } from "constants/styles";
 
+Prism.languages.curl = Prism.languages.js;
+Prism.languages.sh = Prism.languages.bash;
+
+const LIGHT_PALETTE = {
+  light: PALETTE.lightGrey,
+  lightGrey: PALETTE.lightGrey,
+  green: "#6EBD00",
+  blue: "#657EFF",
+  yellow: "#E69D00",
+};
+const DARK_PALETTE = {
+  light: PALETTE.lightGrey,
+  grey: PALETTE.lightGrey,
+  green: "#ABCC7D",
+  blue: "#A9B7FF",
+  yellow: "#FFDD96",
+};
+
+const RESPONSE_THEME = {
+  key: LIGHT_PALETTE.medium,
+  punctuation: LIGHT_PALETTE.grey,
+  boolean: LIGHT_PALETTE.yellow,
+  string: LIGHT_PALETTE.green,
+  number: LIGHT_PALETTE.blue,
+};
+const CODE_THEME = {
+  punctuation: DARK_PALETTE.grey,
+  property: DARK_PALETTE.grey,
+  builtin: DARK_PALETTE.blue,
+  string: DARK_PALETTE.green,
+  function: DARK_PALETTE.yellow,
+  boolean: DARK_PALETTE.yellow,
+  object: DARK_PALETTE.yellow,
+  comment: DARK_PALETTE.grey,
+  variable: DARK_PALETTE.light,
+  htmlTag: DARK_PALETTE.grey,
+  cssVariable: DARK_PALETTE.yellow,
+  cssSelector: DARK_PALETTE.green,
+};
```

Comment:
Worth highlighting. I revamped how we're applying styles so that we can more easily adjust them in the future. New colors from Charles didn't look good on the code examples, so it made sense to split them apart. We might need to double check these as we add content with new languages.